### DISCUSSION TITLE
Speed up script

### DISCUSSION THREAD

**Discussion by kiatchun** [ThioJoe/YT-Spammer-Purge#212] [Q&A]
How can I speed up the loading process of the script.

**Comment by somebadcode** [comment]
> 

Can you elaborate on what is slow? The startup or when it loads the messages?

**Reply by kiatchun** [reply]
The startup.

**Comment by KendallDoesCoding** [comment]
Well, I don't think this is really a big deal because it's just a couple seconds slow, which you should be able to spare, if you want to use the program and delete all the comments, it will save your time from deleting the comments manually in your YT STUDIO. So yeah, its not a big deal according to me.
